index.html

```diff
@@ -172,7 +217,27 @@ <h3>Professional Art Printing Data</h3>
                 </div>
                 <button type="button" class="project-btn2">See Project</button>
```

Comment:
- [ ] Kindly Fix the font weight to the text for the button to match the Figma design it is important for user experience.
 and add styling: Active, Hover, and Pressed please follow the styling and add them to your button to match the Figma